### PR TITLE
Permissions setup flow tweaks

### PR DESCRIPTION
**Pull request by steventux**
## Context

User testing has shown providers entering the setup flow have difficulty understanding the various steps.

## Changes proposed in this pull request

A bunch of content updates and components changes to make the onboarding flow for permissions setup more succinct and simpler.

This PR also includes updates to the organisations index and details pages as the permissions component design changes affect these too.
See [Trello card](https://trello.com/c/i4aP3WOK/2899-permissions-setup-flow-tweaks) for full list of changes to setup flow.
Also [this card](https://trello.com/c/8wWoSZx9/2897-orgs-related-permissions-tweaks) defines the changes needed for org pages.

### Setup step 1
image

### Step 2
image

### Step 3
image

### Organisations index page

image

### Organisation details page

image

## Link to Trello card

https://trello.com/c/i4aP3WOK/2899-permissions-setup-flow-tweaks
and
https://trello.com/c/8wWoSZx9/2897-orgs-related-permissions-tweaks

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb [hunk 1]
```diff
@@ -89,12 +86,10 @@ def next_step
     def previous_step
       if checking_answers.present?
         [:check]
-      elsif current_step == 'info'
-        [:organisations]
       elsif current_step == 'organisations'
         [:start]
```
Comment (mnacos) [inline app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb:90]: A bit confused about `:start` here, what does this map to?

File: app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb [hunk 2]
```diff
@@ -69,15 +69,12 @@ def permissions_for_persistence
 
     # Steps are
     # 1. provider_relationships
```
Comment (mnacos) [inline app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb:71]: Does this comment need updating (e.g. `:start`, `:organisations`)?

**Review by mnacos — APPROVED** [review]
Code looks good and the flow works. Added a couple of minor comments around updating comments. You may want to reset org permissions on the Heroku app, as I used it to review this PR. If @adamsilver is happy with the designs we're ready to ship this. :rocket:

File: app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb (hunk 1 above)
Comment (steventux) [reply app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb:90]: :+1: Yes I think this is misleading as it implies there is a corresponding step. This case could be removed as the return value is never used  as there is no back link on the orgs step. Will simplify in this PR.

File: app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb (hunk 2 above)
Comment (steventux) [reply app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb:71]: I'll remove `:start` as it's redundant

Comment (steventux) [reply app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb:71]: @mnacos housekeeping commit to remove redundant steps https://github.com/DFE-Digital/apply-for-teacher-training/pull/3222/commits/0c20cdc683867ada716f41c22bd050132202834c

File: app/views/provider_interface/provider_relationship_permissions_setup/setup_permissions.html.erb [hunk 3]
```diff
@@ -10,38 +10,22 @@
...
-          </details>
-        </div>
+      <span class="govuk-caption-l">Set up permissions</span>
+      <h1 class="govuk-heading-l"><%= @permissions_model.training_provider.name %> and <%= @permissions_model.ratifying_provider.name %></h1>
+
+      <div class="govuk-body">
+        <h1 class="govuk-heading-m">Who can view applications?</h1>
+        <ul class="govuk-list govuk-list--spaced govuk-!-margin-bottom-6">
+          <li><%= render CheckIcon.new %><%= @permissions_model.training_provider.name %></li>
+          <li><%= render CheckIcon.new %><%= @permissions_model.ratifying_provider.name %> (can only see courses they ratify)</li>
```
Comment (adamsilver) [inline app/views/provider_interface/provider_relationship_permissions_setup/setup_permissions.html.erb:20]:
```suggestion
          <li><%= render CheckIcon.new %><%= @permissions_model.ratifying_provider.name %> – can only see courses they ratify</li>
```

**Review by adamsilver — CHANGES_REQUESTED** [review]
One tiny tweak to the content given an org can have brackets in the name

**Comment by steventux** [comment]
@adamsilver I've commited your suggestion, if you're happy with it can you approve please?

**Review by adamsilver — APPROVED** [review]

**Pull request merged**